browser/brave_rewards/android/brave_rewards_native_worker.cc

```diff
@@ -562,7 +549,7 @@ void BraveRewardsNativeWorker::OnGetCurrentBalanceReport(
   JNIEnv* env = base::android::AttachCurrentThread();
   if (report) {
     std::vector<double> values;
-    values.push_back(report->grants);
+    values.push_back(0);
```

Comment:
optional : i think we can just remove above.

Reply:
Done. I also had to rename the index constants in `BraveRewardsPanel.java`.